assignments/array-methods.js

```diff
@@ -1,6 +1,6 @@
-// A local community center is holding a fund rasising 5k fun run and has invited 50 small businesses to make a small donation on their behalf for some much needed updates to their facilities.  Each business has assigned a representative to attend the event along with a small donation.
+// // A local community center is holding a fund rasising 5k fun run and has invited 50 small businesses to make a small donation on their behalf for some much needed updates to their facilities.  Each business has assigned a representative to attend the event along with a small donation.
```

Comment:
Great job on this file. The functions pass my tests and log the necessary info. Good job recognizing which array methods create a new array and which one does not.